config/development.json

```diff
@@ -84,6 +84,7 @@
 		"jetpack/connect/remote-install": true,
 		"jetpack/connect/mobile-app-flow": true,
 		"jetpack/happychat": true,
+		"jetpack/iframe/editor": true,
```

Comment:
I'd suggest `jetpack/gutenframe` :), just to make it clear that we are not iframing the editor in general case.

Somewhat unrelated to this PR - I've been thinking about your previous remarks about Calypsoify term being confusing in iframe context (what's its place in all of this). I think we should just start using Gutenframe to describe the whole approach, and completely avoid using Calypsoify in the editor context. The above suggestion could help promote that naming consistency in our codebase.

Reply:
By the same token, Guten-anything is confusing, since Gutenberg is a project now involving many things beyond the editor 🙃 . But I'll disagree and commit to the majority of two here 👍 

> Somewhat unrelated to this PR - I've been thinking about your previous remarks about Calypsoify term being confusing in iframe context (e.g. what's its place in all of this?).

Yeah, Calypsoify just really needs a big audit and rethink at this point (not to mention a full review of its flows, which I'm confident has piles of cracks we don't know about). The right time to do that, though, is after we get Gutenframe (urgh!) in place across the board.

Reply:
Gah! @kwight you make a good point

Reply:
> since Gutenberg is a project now involving many things beyond the editor 🙃

My bet is that we'll be iframing even those things in the future for similar integrations. 😄 
I might be wrong though, in which case Gutenframe might not be so future-proof.